internal/srv/metadata.go

```diff
@@ -14,6 +17,11 @@ import (
 
 // LoadBalancerStatusUpdate updates the state of a load balancer in the metadata service
 func (s Server) LoadBalancerStatusUpdate(ctx context.Context, loadBalancerID gidx.PrefixedID, status *metastatus.LoadBalancerStatus) error {
+	// publish event even if metadata endpoint is not configured
+	if err := s.publishLoadBalancerMetadata(ctx, loadBalancerID, status); err != nil {
+		return err
+	}
+
```

Comment:
I think you'll want to move this publish to the same place where we call the status update. We still want status to be updated if the publish fails.